game/views.py

```diff
@@ -123,8 +133,63 @@ def get(request, game_id):
             'tweets': tweets.request_return_handle(game['name'], DetailPageView.TWEETS_LIMIT),
         }
 
+        if request.user.is_authenticated:
+            context.update({
+                'is_added': game_id in list(request.user.favourites.values_list('game__game_id', flat=True))
```

Comment:
nitpick: Can be queried directly from `UserFavouriteGame` model.

btw seems that check for `is_deleted` is missing here